test/fixtures/instance_template/alias_ip_range/terraform.tfvars

```diff
@@ -0,0 +1 @@
+../../shared/terraform.tfvars
```

Comment:
We shouldn't need tfvars. We can hardcode most values in the example and any outputs of setup are automatically exported as a `TF_VAR_*`, so we should have access to `TF_VAR_ project_id` and `TF_VAR_ service_account` from setup.